server/graphql/v2/mutation/VirtualCardMutations.ts

```diff
@@ -162,28 +162,31 @@ const virtualCardMutations = {
     type: new GraphQLNonNull(VirtualCard),
     args: {
       virtualCard: {
-        type: new GraphQLNonNull(VirtualCardUpdateInput),
-        description: 'Virtual Card being edited its new values',
+        type: new GraphQLNonNull(VirtualCardReferenceInput),
+        description: 'Virtual card reference',
+      },
+      name: {
+        type: GraphQLNonNull(GraphQLString),
+        description: 'Virtual card name',
       },
       assignee: {
-        type: AccountReferenceInput,
-        description: 'The individual account reference to attach the Virtual Card to if desired',
+        type: new GraphQLNonNull(AccountReferenceInput),
+        description: 'Individual account responsible for the card',
+      },
```

Comment:
In another comment, I was asking what happens if these are null and now realize the `GraphQLNonNull`.

I think those arguments should be optional.